internal/core/src/storage/DiskFileManagerImpl.cpp

```diff
@@ -59,6 +59,10 @@ DiskFileManagerImpl::~DiskFileManagerImpl() {
         LocalChunkManagerSingleton::GetInstance().GetChunkManager();
     local_chunk_manager->RemoveDir(GetIndexPathPrefixWithBuildID(
         local_chunk_manager, index_meta_.build_id));
+    local_chunk_manager->RemoveDir(GetTextIndexPathPrefixWithBuildID(
```

Comment:
don't think this fix is working.

DiskFileManager is a singleton and never deconstruct.

We should either:
1. clean the directory on deconstruct the segment
2. using a timer thread in diskfile manager and clean all the segment which periodically when segment is releasd

Reply:
Discussed offline: `DiskFileManagerImpl` is per index, so when index is dropped, it's dropped which cleans up the files.